src/engine/models/mesh_material.cpp

```diff
@@ -82,9 +83,9 @@ u8 MeshMaterial::isInFrustum(Plane *t_frustumPlanes, const Vector3 &position)
         for (u8 y = 0; y < 8 && (boxIn == 0 || boxOut == 0); y++)
         {
             boxCalcTemp.set(
-                boundingBox[y].x + position.x,
-                boundingBox[y].y + position.y,
-                boundingBox[y].z + position.z);
+                boundingBoxObj->getVertices()[y].x + position.x,
```

Comment:
I think that it will be good idea to have `boundingBoxObj->getVertex(y).x` like here
https://github.com/h4570/tyra/blob/bb0ec257b2cd4c9aa317043b47a5f998a92dce24/src/engine/include/models/mesh_material.hpp#L53